volumes-web-app/rockcraft.yaml

```diff
@@ -4,117 +4,133 @@ description: |
   This web app is responsible for allowing the user to manipulate PVCs in their Kubeflow
   cluster. To achieve this it provides a user friendly way to handle the lifecycle of PVC
   objects.
-version: v1.7.0_1
+version: "1.8.0"
```

Comment:
Is there reason to removing `v`? In the description, we are still mention `volumes-web-app:v1.8.0`, is that ok?

Reply:
It's the new convention for all ROCKs in the team to not have the `v`
The description was a typo, good observation